newIDE/app/src/ProjectManager/index.js

```diff
@@ -738,6 +760,27 @@ export default class ProjectManager extends React.Component<Props, State> {
                 )
             }
           />
+          {!!this.state.layoutVariablesDialogOpen && (
+            <VariablesEditorDialog
+              open={this.state.layoutVariablesDialogOpen}
+              variablesContainer={this.getLayoutVariables()}
+              onCancel={() => this._onEditLayoutVariable(false)}
```

Comment:
Careful, if you don't pass something it will be undefined. Always pass all variables to your functions. JavaScript might allow you not to do it, but this will bite you very hard later.

Reply:
Can you explain better? I don't really understand.

Reply:
_onEditLayoutVariable is expecting 2 parameters in your code, but you're passing only one.

Reply:
Ok. I get it now. Thanks for the explanation :). Looking to add a default parameter to the _onEditLayoutVariable in cases any of the parameters are not defined. What do you think?

Reply:
I think we can all have in a single variable, using null to represents something not defined.